hasher-matcher-actioner/terraform/api/main.tf

```diff
@@ -223,14 +160,14 @@ resource "aws_apigatewayv2_integration" "hma_apigateway" {
 
 resource "aws_apigatewayv2_authorizer" "hma_apigateway" {
   api_id           = aws_apigatewayv2_api.hma_apigateway.id
-  authorizer_type  = "REQUEST"
-  authorizer_credentials_arn    = aws_iam_role.hma_apigateway.arn
-  authorizer_uri   = aws_lambda_function.api_auth.invoke_arn
-  identity_sources = ["$request.querystring.access_token"]
-  authorizer_payload_format_version = "2.0"
-  enable_simple_responses = true
-  authorizer_result_ttl_in_seconds = 0
-  name             = "${aws_apigatewayv2_api.hma_apigateway.name}_authorizer"
+  authorizer_type  = "JWT"
+  identity_sources = ["$request.header.Authorization"]
+  name             = "${var.prefix}-jwt-authorizer"
+
+  jwt_configuration {
+    audience = ["1hu072n2n920b7kadllfi3imko"]
```

Comment:
did you mean to pass the variables all the way down to here?

Reply:
Doh! Yes I set up all those nice variables and then didn't use them!!!!

Reply:
Fixed!